bindings/rust/s2n-tls/src/callbacks.rs

```diff
@@ -67,17 +67,17 @@ where
 ///
 /// The implementation should verify the certificate host name and return `true`
 /// if the name is valid, `false` otherwise.
-pub trait VerifyHostNameCallback {
+pub trait VerifyHostNameCallback: 'static + Send + Sync {
```

Comment:
I'm rusty on my rust-- why does this need static too?

Reply:
It just means that it can't be borrowing anything that is less than 'static. Since we take ownership of the value, it would be unsound to accidentally extend the lifetime of the borrow.